src/Modal/index.jsx

```diff
@@ -10,7 +10,9 @@ import Icon from '../Icon';
 import newId from '../utils/newId';
 import Variant from '../utils/constants';
 
-const closeModalButtonId = 'paragonCloseModalButton';
+const closeModalButtonId = newId('paragonCloseModalButton');
```

Comment:
@reillz10 to distinguish this from the ```headerCloseModalButtonId``` it may be worth updating to ```footerCloseModalButtonId```, or moving to classes so it doesn't matter the placement of the close button, just that it is a close button.